### PR TITLE
Book Store setup

### PR DESCRIPTION
**Pull request by theBwoyProgrammer**
# UI setup
- Added a form for the inputs
- Added a hard-coded book to display the comment, edit and remove

### CODE REVIEW

File: src/components/Books.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      <small className="category">Musical</small>
+      <h3 className="bookTitle">Trap Diaries</h3>
+      <small className="bookAuthor">Hamxi3bwoy</small>
+
+      <ul className="ul">
+        <li>Comments</li>
+        <li>Remove</li>
+        <li>Edit</li>
+      </ul>
+    </section>
```
Comment (muneebulrehman) [inline src/components/Books.js]: As per the project requirements, you need to create a Book component for displaying a single book that receives a title and an author as prop. Also, you need to reuse it in a component that displays a list of books. Now, this part is super important. Remember, react is all about reusability of components we create.

File: src/components/Books.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    <section className="addBookSec">
+      <h2>ADD BOOK</h2>
+      <form>
+        <input type="text" name="book" value="Title" className="input1" />
+
+        <input type="text" name="author" value="Author" className="input2" />
+
+        <input type="submit" value="Submit" className="input3" />
+      </form>
+    </section>
```
Comment (muneebulrehman) [inline src/components/Books.js]: Also, as per the project requirements, you need to create a separate component for the form.
image

File: src/components/Categories.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    <blockquote cite="https://kidadl.com/quotes/top-most-inspirational-math-quotes-to-inspire-your-students-to-see-the-positives">
+      <p>
+        “There should be no such thing as boring mathematics.”
+      </p>
+      <figcaption>— Edsger W. Dijkstra.</figcaption>
+    </blockquote>
+  </section>
+);
+
+export default Categories;
```
Comment (muneebulrehman) [inline src/components/Categories.js]: This page is not displayed when I click the `categories` link in the navbar. Maybe you misspelled it somewhere. 😊
Also,
image

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @HatimDev-HE , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors

## Required Changes ♻️

- [ ] Kindly add the README file for this project. It is super important to have a custom readme file. Refer to [this](https://github.com/microverseinc/readme-template/blob/master/README.md) template and use it as your base template.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books.js (hunk 1 above)
Comment (theBwoyProgrammer) [reply src/components/Books.js]: ok

**Review by mukhammadrizooff — APPROVED** [review]
Hi @HatimDev-HE, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
-  No linter errors ✔️ 
- Good implementaions ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**